lib/markdown-it/markdown-it-radio-button.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(md, options) {
         if (group) {
           group = group[1];
         } else {
-          group = Math.random().toString(36).substr(2, 5); // generate a new group id
+          group = md5(tokens[i-5].content + tokens[i-4].content + tokens[i].content).substr(2, 5); // generate a deterministic group id
```

Comment:
If you use Crypto library, you can do hash.update(tokens[i-5].content).update(tokens[i-4].content).update(tokens[i].content).digest('hex');